backend/molgenis-emx2-tasks/src/main/java/org/molgenis/emx2/tasks/ScriptTask.java

```diff
@@ -151,6 +156,14 @@ public void run() {
     } catch (Exception e) {
       this.setError("Script failed: " + e.getMessage());
       throw new MolgenisException("Script execution failed", e);
+    } finally {
+      if (getStatus() == TaskStatus.ERROR) {
+        EmailSettings emailSettings = new EmailSettings.EmailSettingsBuilder().build();
```

Comment:
Is creation of emailService expensive? If so, the we should create one at class level, otherwise okay here.